docs/advanced_topics/customisation/custom_bulk_actions.rst

```diff
@@ -0,0 +1,123 @@
+Adding custom bulk actions
+==========================================
+
+This document describes how to add custom bulk actions to different listings
+
+
+Registering a custom bulk action
+--------------------------------
+
+    .. code-block:: python
+
+        from wagtail.admin.views.bulk_action import BulkAction
+        from wagtail.core import hooks
+
+
+        @hooks.register('register_bulk_action')
+        class CustomPageBulkAction(BulkAction):
+            display_name = _("Custom Action")
+            aria_label = _("Do custom action")
+            action_type = "action"
+            template_name = "/path/to/template"
```

Comment:
It would be good to provide an example of the template rather than have the very generic "path/to/template"